### ISSUE TITLE
Old quotes are a bad representation of current culture

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5674]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#5674|
|Type|defect|
|Created|2012-05-09 19:29:22Z|
|Branch|https://github.com/twisted/twisted/tree/old-quotes-5674-2|

The historical quote file is currently in ascending order, starting with the oldest quotes. Some of the quotes (though nothing past 2006 AFAICT) are offensive, e.g. sexist, and it is not clear to the casual reader that these are no longer reflective of our culture.

As a solution I propose splitting up the file, changing the order to be descending (from new to old), and adding a note/apology to that effect.

<details><summary>Searchable metadata</summary>

```
trac-id__5674 5674
type__defect defect
reporter__itamar itamar
priority__normal normal
milestone__ 
branch__branches_old_quotes_5674_2 branches/old-quotes-5674-2
branch_author__itamarst__itamar itamarst, itamar
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1336591762000000 1336591762000000
changetime__1344790093000000 1344790093000000
version__None None
owner__itamar itamar

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph commented|
|-|-|

I like the idea.

My preference would be to divide up the quote file by release version, since new versions will be divided up that way, rather than by year, which is arbitrary and doesn't really correspond to any milestones in Twisted development or history.

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_2"></a>|@itamarst commented|
|-|-|

(In [34365]) Branching to 'old-quotes-5674'

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_3"></a>|@itamarst commented|
|-|-|

I suppose splitting by release makes changing the order less necessary, since each file will cover a smaller timespan. So maybe I'll just do that.

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_4"></a>|@itamarst set owner to @glyph|
|-|-|

I split up quotes by release, added an explanation and apology, and excluded doc/historic/ from the release tarball. No reason to include either old quotes or talks from 2002 in our every release tarball; if anyone cares they can do a checkout or use trac to read it.

http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/old-quotes-5674

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to @itamarst|
|-|-|

Looks pretty good to me.  My only thought is that we might want to have some more developers sign the README.

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_6"></a>|@itamarst commented|
|-|-|

Shall I add you, JP? How about Glyph? (It's also easy to add one's name later, since really historic/ isn't under code review process either).

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_7"></a>|@glyph commented|
|-|-|

Replying to [itamar](#note_6):
> Shall I add you, JP? How about Glyph? (It's also easy to add one's name later, since really historic/ isn't under code review process either).

Go ahead and add me.  And let's get this over with :).

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_8"></a>|@exarkun commented|
|-|-|

(In [34646]) Pronoun adjustment and additional signatures

refs [#5674](https://github.com/twisted/twisted/issues/5674)

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_9"></a>|@itamarst commented|
|-|-|

(In [35153]) Branching to 'old-quotes-5674-2'

**Comment by twisted-trac** [comment]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)<a name="note_10"></a>|@itamarst set status to `closed`|
|-|-|

(In [35155]) Merge old-quotes-5674-2: Deal with unfortunate old quotes.

Author: itamar
Review: exarkun
Fixes: [#5674](https://github.com/twisted/twisted/issues/5674)
Deal with old quotes: split up by release, added apology for the bad ones. Additionally, doc/historic/ is no longer packaged in the tarball.
